src/registrar/migrations/0124_delete_cascade_submitter_contacts.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations
```

Comment:
Deletes any Contact objects that only exist to be submitter contacts for domain requests and domain information